module/input/ImageDecompressor/src/ImageDecompressor.cpp

```diff
@@ -9,6 +9,8 @@
 #include "message/input/Image.hpp"
 #include "message/output/CompressedImage.hpp"
 
+#include "utility/support/yaml_LogLevel.hpp"
```

Comment:
this filename is an abomination. it either needs to be snake_case or something else.